src/rix.cpp

```diff
@@ -159,7 +159,12 @@ void CrixPlayer::rewind(int subsong)
   data_initial();
 }
 
+#ifdef DJGPP
```

Comment:
looking at player.h, it appears that it should be `unsigned int getsubsongs()` for all systems, not just DJGPP